src/app/@ansyn/ansyn/modules/plugins/cesium/plugins/annotations.visualizer.ts

```diff
@@ -0,0 +1,152 @@
+					...activeAnnotationLayer.data,
+					features: activeAnnotationLayer.data.features.concat(GeoJSON.features),
+				};
+				// TODO - display overlays with cesium
+				// if (this.overlay) {
+				// 	GeoJSON.features[0].properties = {
+				// 		...GeoJSON.features[0].properties,
+				// 		...this.projectionService.getProjectionProperties(this.communicator, data, feature, this.overlay)
+				// 	};
+				// }
+				this.store.dispatch(
+					new UpdateLayer({
+						id: activeAnnotationLayer.id,
+						data,
+					})
+				);
+			})
+		);
+
+	constructor(protected actions: Actions, public loggerService: LoggerService, public store: Store<any>) {
+		super();
+	}
+
+	renderEntities(
+		entities: {
+			[key: string]: ILayer;
+		},
+		isAnnotationSubMenuOpen: boolean,
+		selectedLayersIds: string[],
+		isActiveMap: boolean,
+		activeAnnotationLayer: string
+	): Observable<boolean> {
+		const displayedIds = uniq(
+			isActiveMap && isAnnotationSubMenuOpen ? [...selectedLayersIds, activeAnnotationLayer] : [...selectedLayersIds]
+		).filter((id: string) => entities[id] && entities[id].type === LayerType.annotation);
+
+		const features = displayedIds.reduce((array, layerId) => [...array, ...entities[layerId].data.features], []);
+		return this.showAnnotation(featureCollection(features) as FeatureCollection);
+	}
+
+	showAnnotation(annotationsLayer: FeatureCollection<any>): Observable<boolean> {
+		const annotationsLayerEntities = this.annotationsLayerToEntities(annotationsLayer);
+		this.getEntities()
+			.filter(({ id }) => !annotationsLayerEntities.some((entity) => id === entity.id))
+			.forEach(({ id }) => this.removeEntity(id));
+
+		const entitiesToAdd = annotationsLayerEntities.filter((entity) => {
+			const oldEntity = this.idToEntity.get(entity.id);
+			if (oldEntity) {
+				const isShowMeasuresDiff = oldEntity.originalEntity.showMeasures !== entity.showMeasures;
+				const isShowAreaDiff = oldEntity.originalEntity.showArea !== entity.showArea;
+				const isLabelDiff = oldEntity.originalEntity.label !== entity.label;
+				const isFillDiff = oldEntity.originalEntity.style.initial.fill !== entity.style.initial.fill;
+				const isStrokeWidthDiff = oldEntity.originalEntity.style.initial["stroke-width"] !== entity.style.initial["stroke-width"];
+				const isStrokeDiff = oldEntity.originalEntity.style.initial["stroke"] !== entity.style.initial["stroke"];
+				const isOpacityDiff = ["fill-opacity", "stroke-opacity"].filter(
+					(o) => oldEntity.originalEntity.style.initial[o] !== entity.style.initial[o]
+				);
+				return isShowMeasuresDiff || isLabelDiff || isFillDiff || isStrokeWidthDiff || isStrokeDiff || isOpacityDiff || isShowAreaDiff;
+			}
+			return true;
+		});
+		return this.addOrUpdateEntities(entitiesToAdd);
+	}
+
+	onInit() {
+		super.onInit();
+		this.cesiumDrawer = this.communicator.getPlugin(CesiumDrawAnnotationsVisualizer);
+	}
+
+	onResetView(): Observable<boolean> {
+		return of(true);
+	}
+}
```

Comment:
make sure that imagery-submodules will reference the master branch

Reply:
currently it should reference angular9 branch